dev/app/builtin/init-stories.js

```diff
@@ -1,6 +1,6 @@
 import initAnalyticsStories from './stories/analytics.stories';
 import initBreadcrumbStories from './stories/breadcrumb.stories.js';
-import initClearAllStories from './stories/clear-all.stories';
+import initClearRefinementsStories from './stories/clear-refinements.stories.js';
```

Comment:
Oh yes agreed. By default, VScode will not add it but that's not the behavior everywhere. Most of the code in is.js has `.js` extensions, but with a rule it can be `--fix`ed easily. No opinions on the topic, happy to do the changes :)

Reply:
Well `--fix` doesn't work for the rule: `import/extensions`. This makes this change very painful. Also the codebase is mostly using extension except for packages.

Reply:
But we can enforce the extensions only for InstantSearch but at least statute on something.

Reply:
> But we can enforce the extensions only for InstantSearch but at least statute on something.

Ok. If I were to make a choice here, I'd go for no extensions:
 - it's easier to setup the rules in eslint
 - we are inconsistent when it comes to packages (no extensions even for files in packages)
 - vsCode (the editor that we use as far I understand) doesn't add extensions by default

It's a big change but I guess it could be done with a mass replace.

Reply:
https://github.com/algolia/instantsearch.js/pull/3022